src/Components/OneTrustConsentStyles.ts

```diff
@@ -412,6 +412,8 @@ export const OneTrustConsentStyles = createGlobalStyle`
       ${toStyle({
         boxShadow: DROP_SHADOW,
         borderTop: `1px solid ${THEME.colors.black15}`,
+        // keeping the banner below the onboarding modal
+        zIndex: "999 !important",
```

Comment:
So: this `toStyle` function automatically `!important`s everything in it. Also: we _want_ to keep this below the modal? It seems to me that this should be above everything.

Reply:
Also, you probably know this but you're next steps didn't list it. Changing this file doesn't actually change the OneTrust stuff. The remaining steps need to be followed:

```
* - in Boot.tsx:
 * `import { OneTrustConsentStyles } from "Components/OneTrustConsentStyles"`
 * and include the `<OneTrustConsentStyles />` component in the render method.
 *
 * When you're ready, you can just return only the `<OneTrustConsentStyles />`
 * component from Boot.tsx, then copy out the compiled styles. Log into
 * onetrust.com and find the template. Click on "Styling", then paste into the
 * "Custom CSS" field. Save the template, then go to "Integration" > "Scripts".
 * Click the environment and publish.
```

Reply:
I suspect a much lower value is sufficient. We avoid a complex z-index hierarchy, generally speaking. You'll notice that we only use the values `0`, `1` and very rarely: `2`. Things tend to have a natural stacking order and the Modal is a legacy oddity.

Reply:
Though now that I remember it, I did extract a Z index object for some of the weird ones. Check `Apps/Components/constants` and extract these into those.

Reply:
roger! i was following the modal's 9999 -- will mod

Reply:
(will add the ontrust stuff to the PR desc too)

Reply:
looking at z-indices from `constants`: the footer is 1, so the consent banner should be 1, and the zendesk widget 0. will push in a bit